signwriting_transcription/pose_to_signwriting/bin.py

```diff
@@ -32,8 +32,8 @@ def main():
     args = get_args()
     print('Downloading model...')
     os.makedirs("experiment", exist_ok=True)
```

Comment:
also - need to check if the model exists already, no need to download every single time